robot/Cumulus/resources/NPSP.py

```diff
@@ -1208,23 +1224,25 @@ def verify_record_is_created_in_database(self,object_name,id):
     @capture_screenshot_on_error
     def select_value_from_dropdown(self,dropdown,value):
         """Select given value in the dropdown field"""
-
-        if dropdown in ("Open Ended Status","Payment Method") and self.latest_api_version == 50.0:
-            locator =  npsp_lex_locators['record']['rdlist'].format(dropdown)
-            selection_value = npsp_lex_locators["erd"]["modal_selection_value"].format(value)
-            if self.npsp.check_if_element_exists(locator):
-                self.selenium.set_focus_to_element(locator)
-                self.selenium.wait_until_element_is_visible(locator)
-                self.selenium.scroll_element_into_view(locator)
-                self.salesforce._jsclick(locator)
-                self.selenium.wait_until_element_is_visible(selection_value)
-                self.selenium.click_element(selection_value)
+        if self.latest_api_version == 51.0:
+            self.click_flexipage_dropdown(dropdown,value)
         else:
-            locator = npsp_lex_locators['record']['list'].format(dropdown)
-            self.selenium.scroll_element_into_view(locator)
-            self.selenium.get_webelement(locator).click()
-            self.wait_for_locator('popup')
-            self.npsp.click_link_with_text(value)
+            if dropdown in ("Open Ended Status","Payment Method") and self.latest_api_version == 50.0:
```

Comment:
Do we need another check here for api version ? Given the values are in "either or" condition.

Reply:
good catch, updated